apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/use-global-styles-config.js

```diff
@@ -0,0 +1,32 @@
+import { useSelect } from '@wordpress/data';
+
+export function useGlobalStylesConfig() {
+	return useSelect( ( select ) => {
+		const {
+			getEditedEntityRecord,
+			__experimentalGetCurrentGlobalStylesId,
+			__experimentalGetDirtyEntityRecords,
+		} = select( 'core' );
+
+		const _globalStylesId = __experimentalGetCurrentGlobalStylesId
+			? __experimentalGetCurrentGlobalStylesId()
+			: null;
+		const globalStylesRecord = getEditedEntityRecord( 'root', 'globalStyles', _globalStylesId );
+
+		const globalStylesConfig = {
+			styles: globalStylesRecord?.styles ?? {},
+			settings: globalStylesRecord?.settings ?? {},
+		};
+
+		// Do not show the notice if the use is trying to save the default styles.
```

Comment:
This comment should be more generic, since `useGlobalStylesConfig` doesn't know whether `isVisible` is being used for a notice.

Actually, I think this property should be called `globalStylesInUse` or something like that, since that's really the information we're returning here.

Reply:
Good idea, done! :)